app/views/vacancies/index.html.slim

```diff
@@ -28,6 +28,17 @@ h1 class="govuk-heading-l" role="heading" aria-level="1"
 
       #search-results
         - if any_vacancies
+          div class="sort-container"
+            div class="left"
+              - sort_by_value = @vacancies_search.sort.options.find { |option| option.by == @vacancies_search.sort.by }.display_name.downcase
+              - sorted_by_string = t("jobs.sort_by.jobs_page_label", sort_by_value: sort_by_value)
+              h2 class="govuk-heading-m govuk-!-margin-bottom-3 govuk-!-margin-top-1" = sorted_by_string
+            div class="right"
```

Comment:
Non-blocking suggestion: No expert on CSS but I suspect that using classes that, since these `left` and `right` classes are not bringing any CSS styling, either they're redundant or we want to switch them to `id=` if they're used as identifiers.